docs/Contributing/API-for-contributors.md

```diff
@@ -671,19 +689,265 @@ Content-Type: application/octet-stream
 
 `Status: 204`
 
+### Update ABM token's teams
+
+`PATCH /api/v1/fleet/abm_tokens/:id/teams`
 
-### Disable automatic enrollment (ADE)
+#### Parameters
 
-`DELETE /api/v1/fleet/mdm/apple/abm_token`
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id | integer | path | *Required* The ABM token's ID |
+| macos_team_id | integer | body | macOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ios_team_id | integer | body | iOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ipados_team_id | integer | body | iPadOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
 
 #### Example
 
-`DELETE /api/v1/fleet/mdm/apple/abm_token`
+`PATCH /api/v1/fleet/abm_tokens/1/teams`
+
+##### Request body
+
+```json
+{
+  "macos_team_id": 1,
+  "ios_team_id": 2,
+  "ipados_team_id": 3,
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+"abm_token": {
+  "id": 1,
+  "apple_id": "apple@example.com",
+  "org_name": "Fleet Device Management Inc.",
+  "mdm_server_url": "https://example.com/mdm/apple/mdm",
+  "renew_date": "2024-11-29T00:00:00Z",
+  "terms_expired": false,
+  "macos_team": "Workstations",
+  "ios_team": "Company-owned iPhones",
+  "ipados_team": "Company-owned iPads"
```

Comment:
These should be team IDs  (numbers & not strings) since that's what's in the request

```suggestion
  "macos_team": 1,
  "ios_team": 2,
  "ipados_team": 3
```

Reply:
@jahzielv just to confirm, we still ask for team names (not IDs) in the YAML files right? 

https://github.com/fleetdm/fleet/pull/21043/files#diff-b71104232d8fbaaf4bd537065533cbf39ec5f9bf028ec1d8083346953ed5178dR602

Reply:
@noahtalerman yes, still using team names in YAML 👍